### DISCUSSION TITLE
How do I create a theme for e107?

### DISCUSSION THREAD

**Discussion by jpjoneshy1** [e107inc/e107#5276] [Q&A Themes]
I used to use e107 back in the v1x days and had taught my self how to create a theme, but it's all different now.  I there any detailed documentation on how to create a theme?

Thanks,
JP

**Comment by Jimmi08** [comment]
https://devguide.e107.org/theme-development/introduction

And study the core bootstrap5 theme.  Not bootstrap3 - it is admin theme and it is more complicated. _blank theme is outdated too.

**Comment by jpjoneshy1** [comment]
Thanks.  Have you been able to mess around with the e107/TNG thing?  I sent
you an email asking about it.

 

JP

**Reply by Jimmi08** [reply]
Hi, I have no email related to TNG.  Sorry.

**Comment by jpjoneshy1** [comment]
Ok, no problem.  But have you ever tried to integrate e107 and tng?  We
talked about it briefly last year but you were heading out to holiday.

**Reply by Jimmi08** [reply]
It is not free software.   
Do you mean this discussion?
https://github.com/e107inc/e107/discussions/5047

**Comment by Jimmi08** [comment]
I just found my old docs - ignore Ayro stuff .  Maybe some information are still useful. 

[e107 + AYRO UI - version 1 2022.pdf](https://github.com/user-attachments/files/15983470/e107.%2B.AYRO.UI.-.version.1.2022.pdf)
